server/middleware/roleBasedAccessControl.ts

```diff
@@ -1,15 +1,76 @@
 import authorisationMiddleware from './authorisationMiddleware'
+import ApplicationRole from '../enums/applicationRole'
+import ApplicationAction from '../enums/applicationAction'
 
-enum ApplicationRoles {
-  ROLE_EDUCATION_WORK_PLAN_EDITOR = 'ROLE_EDUCATION_WORK_PLAN_EDITOR',
-}
+/**
+ * Deprecated - use `checkUserHasPermissionTo` instead
+ */
+const checkUserHasEditAuthority = () => authorisationMiddleware([ApplicationRole.ROLE_EDUCATION_WORK_PLAN_EDITOR])
```

Comment:
`checkUserHasEditAuthority` will be nuked once we've migrated everything over to the new `checkUserHasPermissionTo` function
But at the moment it is used in a lot of places, so we need to keep it for now